src/components/Announcement.re

```diff
@@ -0,0 +1,59 @@
+open Globals;
+open Css;
```

Comment:
I typically don't usually open Css globaly, but I guess it is fine if it doesn't cause name conflicts